architecture/topics/router_environment_variables.adoc

```diff
@@ -61,6 +61,7 @@ and "-". The default is the hashed internal key name for the route.
 |`ROUTER_SYSLOG_ADDRESS` |  | Address to send log messages. Disabled if empty.
 |`ROUTER_SYSLOG_FORMAT` | | If set, override the default log format used by underlying router implementation. Its value should conform with underlying router implementation's specification.
 |`ROUTER_TCP_BALANCE_SCHEME` | source | xref:load-balancing[Load-balancing strategy] for multiple endpoints for pass-through routes. Available options are `source`, `roundrobin`, or `leastconn`.
+|`ROUTER_THREADS` | | Specifies the number of threads for the haproxy router.
```

Comment:
'HAProxy router' instead of 'haproxy router', just to make it consistent with rest of the doc?